content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -23,3 +23,118 @@ All students need to provably understand all of the following concepts:
 - getters and setters
 - inheritance and overriding and extending methods
 - composition
+
+# Python
+
+## Constructors
+The `__init__()` method is called the constructor and is always called when an object is created, generally used for instantiating an object.
+
+### Default Constructor
+A simple constructor which doesn’t accept any arguments. Its definition has only one argument which is a reference to the instance being constructed.
+```
+class BankAccount:
+  def __init__(self):
+    pass
+```
+
+### Parameterized Constructor
+A constructor with parameters. Takes its first argument as a reference to the instance being constructed known as self and the rest of the arguments are provided by the programmer.
```

Comment:
I'd say:
As the name suggests, a parameterised constructor, is a constructor with additional parameters. Takes its first argument as a reference to the instance being constructed, known as 'self', and the rest of the arguments are provided by the programmer.